### PR TITLE
Add cuttle cask

### PR DESCRIPTION
**Pull request by LesleyMdCampo**

### CODE REVIEW

**Comment by shaunlebron** [comment]
@LesleyMdCampo looks like we need an empty line at the end of the file for the test to pass

File: Casks/cuttle.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'cuttle' do
+  version '1.1'
+  sha256 '7133beb24c3a37fe13f12d0b8110f1fcaf541d80e938cfaafb37a4d47bfcca15'
+
+  url 'https://github.com/oakmac/cuttle/releases/download/v1.1/cuttle-v1.1-mac.dmg'
```
Comment (fanquake) [inline Casks/cuttle.rb]: Can you use #{version} here. i.e download/v#{version}/cuttle-v#{version}-mac.dmg'

**Comment by fanquake** [comment]
Once you've made the changes can you please also squash your commits?

**Comment by LesleyMdCampo** [comment]
@fanquake - I'm not sure what the error in the Travis CI build is here. Can you help?

File: Casks/cuttle.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'cuttle' do
+  version '1.1'
+  sha256 '7133beb24c3a37fe13f12d0b8110f1fcaf541d80e938cfaafb37a4d47bfcca15'
+
+  url 'https://github.com/oakmac/cuttle/releases/download/v#{version}/cuttle-v#{version}-mac.dmg'
```
Comment (vitorgalvao) [inline Casks/cuttle.rb]: The error is here. You’re using interpolation (`#{version}`), so you need to use double quotes, or else it’s taking the whole string literally.

**Comment by fanquake** [comment]
@LesleyMdCampo Thanks for updating, looks good now.

**Pull request merged**